### PR TITLE
add glossary terms

### PR DESCRIPTION
**Pull request by micheleRP**
fixes https://github.com/redpanda-data/documentation-private/issues/841

### CODE REVIEW

File: modules/terms/partials/seastar.adoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+=== Seastar
+:term-name: Seastar
+:hover-text: An open-source thread-per-core C++ framework, which binds all work to physical cores. Redpanda is built on Seastar.
+:link: https://seastar.io/
```
Comment (Deflaimun) [inline modules/terms/partials/seastar.adoc]: @JakeSCahill do we need to do something different for external links?

**Review by Deflaimun — APPROVED** [review]
lgtm. Left one comment about external links

File: modules/terms/partials/acl.adoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+=== access-control list (ACL) 
+:term-name: ACL
+:hover-text: A security feature used to define and enforce granular permissions to resources, ensuring only authorized users or applications can perform specific operations. ACLs act on principals. 
+:link: https://docs.redpanda.com/current/manage/security/authorization/
```
Comment (micheleRP) [inline modules/terms/partials/acl.adoc]: @JakeSCahill also not sure if this is how to add internal links.

Comment (micheleRP) [reply modules/terms/partials/acl.adoc]: I want these as see also type links in the glossary page (not a direct link like in the [existing cert-manger term](https://docs.redpanda.com/current/manage/kubernetes/security/tls/use-cert-manager/). Should I remove internal links from the `:link:` attribute and just put them as See also text under the hover text?

Comment (JakeSCahill) [reply modules/terms/partials/acl.adoc]: Yes 👍 The default behavior links to the glossary page. The purpose of the `link` attribute is to override that behavior to link to an external site for the definition.

File: modules/terms/partials/community.adoc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+=== Redpanda Community Edition
+:term-name: Redpanda Community Edition
+:hover-text: A license with the Redpanda Business Source License (BSL). These core features are free and source-available.
```
Comment (JakeSCahill) [inline modules/terms/partials/community.adoc]:
```suggestion
:hover-text: Redpanda software that is available under the Redpanda Business Source License (BSL). These core features are free and source-available.
```

File: modules/terms/partials/admin-api.adoc [hunk 4]
```diff
@@ -0,0 +1,7 @@
+=== Admin API
+:term-name: Admin API
+:hover-text: A REST API used to manage and monitor Redpanda clusters. It uses the default port 9644.
+
+For more information, see https://docs.redpanda.com/api/admin-api/[Redpanda Admin API].
```
Comment (JakeSCahill) [inline modules/terms/partials/admin-api.adoc]: We should be using the usual xrefs for internal links

**Comment by JakeSCahill** [comment]
We need to delete the test terms too.

**Pull request merged**